app/views/dashboard.pug

```diff
@@ -20,19 +20,19 @@ html(lang="en")
                 h1 Welcome to Hive!
 
         // Side Button Bar
-        div(id='navBar' style='position:absolute;width:10%;height:100%;left:0%;top:0%;border-style:solid;background-color:rgb(49,60,80);')
+        nav(class="navbar navbar-nav" id='navBar' style='position:absolute;width:10%;height:100%;left:0%;top:0%;border-style:solid;background-color:rgb(49,60,80);')
 
-            div(style='position:absolute;width:100%;height:20%;left:0%;top:0%;')
-                input(id='home' style='width:100%; height:100%;' type='image' src="/assets/hive-logo2.png" alt='Submit')
+            li(class='active' style='position:absolute;width:100%;height:20%;left:0%;top:0%; padding-top:25px; padding-bottom:25px' )
+                input(id='home' style='width:100%; height:100%;' type='image' src="/assets/hive.png" alt='Submit' href='#')

```

Comment:
it's suppose to pass the active class, it didn't really make sense to me but that is what I found via bootstrap tutorial, it didn't work anyway so I'll just get rid of those.